config/examples/Creality/Ender-3 Pro/CrealityV422/Configuration_adv.h

```diff
@@ -583,7 +583,7 @@
  *
  * Define one or both of these to override the default 0-255 range.
  */
-#define FAN_MIN_PWM 50
+//#define FAN_MIN_PWM 50
```

Comment:
Pretty much *every* ender has this commented out so makes sense to align I guess.  

In some of my tests with lasers using FAN connector, I'd notice the stock fan didn't reliably startup somewhere around 20 PWM.  I was sometimes being lazy and testing new laser features by watching my FAN movement so that 20 is a rough guess... But I'm guessing the original 50 did likely come from someone noticing an issue at some point.

Reply:
I'm not sure if any of the fans have PWM? Both of the fans on the hot end are two wire fans, so no RPM coming back, and I've never seen or heard a fan change RPM. I'll dig a little deeper and see if the setting is even relevant for the Ender-3 Pro.

Reply:
Yes the part cooling fan is definitely PWM controllable. You may need `FAST_PWM_FAN` and a frequency around 50 to 100 or `FAN_SOFT_PWM` with `SOFT_PWM_SCALE` of 1 or so to actually control it though, depending on the fan.

Reply:
Parts cooling fan is adjustable via PWM and a value of 50-65 is usually the minimum PWM required to get it spinning when selecting 1% fan speed.

Reply:
Then I'll run a few tests first to see how I go about configuring fan PWM.